app/controllers/api/user_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UseController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
I think this should be in `ApiController` instead, so that it will require authentications per API calls